pkg/controller/core.oam.dev/v1alpha2/application/apply.go

```diff
@@ -75,15 +70,14 @@ func readyCondition(tpy string) runtimev1alpha1.Condition {
 }
 
 type appHandler struct {
-	r                        *Reconciler
-	app                      *v1beta1.Application
-	appfile                  *appfile.Appfile
-	inplace                  bool
-	isNewRevision            bool
-	revisionHash             string
-	acrossNamespaceResources []v1beta1.TypedReference
-	resourceTracker          *v1beta1.ResourceTracker
```

Comment:
Why these two not needed now?

Reply:
dispatch will handle tracking resources, no matter it's cluster-scope or cx-namspace. So we don't need to handle it in app controller.